### PR TITLE
Add ALDImobile

### PR DESCRIPTION
**Pull request by fletcherdodd**
Was unable to find a suitable SVG logo so had to go with a favicon PNG

### CODE REVIEW

**Comment by phallobst** [comment]
There you go:

File: entries/a/aldimobile.com.au.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "ALDImobile": {
+    "domain": "aldimobile.com.au",
+    "img": "aldimobile.com.au.png",
```
Comment (Carlgo11) [inline entries/a/aldimobile.com.au.json]:
```suggestion
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
As requested by @phallobst, please replace the icon with the suggested SVG:
```SVG
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 26 26"><defs><radialGradient id="a" cx="13" cy="13" r="13" fx="12" fy="13" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#000050"/><stop offset=".63" stop-color="#010151"/><stop offset=".78" stop-color="#040352a5"/><stop offset=".85" stop-color="#03025267"/><stop offset=".93" stop-color="#02025121"/><stop offset="1" stop-color="#01015100"/></radialGradient><filter id="b" width="1.3" height="1.3" x="-.15" y="-.15"><feGaussianBlur/></filter></defs><circle cx="13" cy="13" r="10" fill="none" stroke="url(#a)" stroke-width="6" filter="url(#b)"/></svg>
```
Thanks! 🙂

**Review by phallobst — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]
Thanks for the pull request @fletcherdodd!
It will be public on 2fa.directory within a few minutes.

**Pull request merged**